tools/setup/setup_venvs.py

```diff
@@ -30,7 +30,8 @@ def main(is_travis=False):
             setup_virtualenv(PY3_VENV_PATH, PY3_DEV_REQS_FILE, patch_activate_script=True,
                              virtualenv_args=['-p', 'python3'])
     else:
-        setup_virtualenv(PY2_VENV_PATH, PY2_DEV_REQS_FILE, patch_activate_script=True)
+        if PY2:
+            setup_virtualenv(PY2_VENV_PATH, PY2_DEV_REQS_FILE, patch_activate_script=True)
```

Comment:
Maybe add a comment here explaining that we're now only updating the Python 2 virtualenv if the user is actually running with Python 2?